src/app/content/components/Toolbar.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { SFC } from 'react';
+import styled from 'styled-components';
+import {ListOl} from 'styled-icons/boxicons-regular/ListOl';
+import {Search} from 'styled-icons/fa-solid/Search';
+import {Print} from 'styled-icons/fa-solid/Print';
+import theme from '../../theme';
+
+const ListIcon = styled(ListOl)`
+  height: 2rem;
+  width: 2rem;
+  color: #027EB5;
+  margin-right: 0.7rem;
+
+  :hover {
+    color: #0064A0;
+  }
+
+`;
+
+const SearchIcon = styled(Search)`
+  height: 2rem;
+  width: 2rem;
+  color: #818181;
+  margin-right: 0.7rem;
```

Comment:
this is the right margin, but because the element is `position: absolute;` its not doing anything. is there a reason for the absolute positioning? the print icon isn't absolute and has good looking margin